zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -48,10 +68,8 @@ pub enum ProofInstruction {
 
     /// Verify a withdraw zero-knowledge proof.
     ///
-    /// This instruction can be configured to optionally initialize a proof context state account.
-    /// If creating a context state account, an account must be pre-allocated to the exact size of
-    /// `ProofContextState<WithdrawProofContext>` and assigned to the ZkToken proof program prior
-    /// to the execution of this instruction.
+    /// This proof is a collection of smallers proofs that are required by the SPL Token 2022
```

Comment:
Oh shoot, I just noticed the "smallers" typo. Let's fix here, but I'll submit a separate PR to master to fix.

Reply:
```suggestion
    /// This proof is a collection of smaller proofs that are required by the SPL Token 2022
```

Reply:
Thanks for the catch!